### ISSUE TITLE
apr 17

### ISSUE THREAD

**Issue by hillarysanders** [hillarysanders/notes#13]
last week:
- [x] develop code to be able to very easily reboot an instance
- [x] added validation-data loss tracking to tensorboard to track alongside training loss
- [x] added timing tracking to tensorboard (seconds per epoch)

today:
- [x] compared mlp to convnet - they're still both going down in validation loss, so they're both still improving. It looks like MLP definitely beats classic convnet, but if you remove all dropout from convnet it is neck and neck with MLP. Neither's validation AUC / Loss has gone down yet, but it seems like convnet's is likely to sooner given the 0 dropout. Also, convnet epochs take about 10X as long than MLP epochs, so that's something to keep in mind (same for testing time???)
- [ ] going to add some more timing tracking if possible... e.g. it seems like GPU and CPU usage will be way down and nothing will be happening, or the epochs will be training super slowly - and I think it has to do with the issues Rich has been talking about with Keras & its memory usage. --> I'm going to hold off on tests surrounding that since he knows more and I don't want to duplicate effort, but definitely important.
- [ ] create a nice system to testing out fitted model on dataset
- [ ] test out current models on Sophos benign data
